List prerequisites (#1133)
mongodb/mongodb-kubernetes-operator: docs/install-upgrade.md

- [Upgrade the Operator](#upgrade-the-operator)
- [Rotating TLS certificate for the MongoDB deployment](#rotating-tls-certificate-for-the-mongodb-deployment)

## Prerequisites

> - A Kubernetes cluster with nodes with x86-64/AMD64 processors (either all, or a separate node pool)

Reviewer: probably should be `amd64` to be aligned with the `GOARCH` env variable.

*`secret_name` is what you've specified under `Spec.Security.TLS.CertificateKeySecret.Name`*.

If you're using a tool like cert-manager, you can follow [these instructions](https://cert-manager.io/docs/usage/certificate/#renewal) to rotate the certificate.

> The operator should would watch the secret change and re-trigger a reconcile process. 

Reviewer 2: ```suggestion
The operator will watch the secret change and re-trigger a reconcile process. 
```
?
